server/statistics/store.go

```diff
@@ -311,10 +311,10 @@ const (
 // NewRollingStoreStats creates a RollingStoreStats.
 func newRollingStoreStats() *RollingStoreStats {
 	return &RollingStoreStats{
-		bytesWriteRate:          NewTimeMedian(DefaultAotSize, DefaultWriteMfSize),
-		bytesReadRate:           NewTimeMedian(DefaultAotSize, DefaultReadMfSize),
-		keysWriteRate:           NewTimeMedian(DefaultAotSize, DefaultWriteMfSize),
-		keysReadRate:            NewTimeMedian(DefaultAotSize, DefaultReadMfSize),
```

Comment:
Can we keep this and make `interval` an optional arg?

Reply:
store heartbeat interval and region heartbeat interval is different